config.toml

```diff
@@ -79,6 +79,18 @@ tx_confirmation = false
 # [Default: false]
 auto_register_counterparty_payee = false
 
+# Set the maximum size for the memo field in ICS20 packets.
+# If the size of the memo field is bigger than the configured
+# one, the packet will not be relayed.
+# [Default: "32KiB"]
+#ics20_max_memo_size = "32KiB"
+
+# Set the maximum size for the receiver field in ICS20 packets.
+# If the size of the receiver field is bigger than the configured
+# one, the packet will not be relayed.
+# [Default: "2KiB"]
+#ics20_max_receiver_size = "2KiB"
```

Comment:
Can you add config on how to achieve old behaviour, i.e. no checks?

Reply:
Good point, I'll slightly change the configuration to have something along the lines of:

`ics20_max_receiver_limit = { enabled = true, size = "2KiB" }`
`ics20_max_memo_limit = { enabled = true, size = "32KiB" }`